### ISSUE TITLE
Heroku 결제 문제 

### ISSUE THREAD

**Issue by plainOldCode** [BIYN-kr/biyn-web-static#13]
Heroky Hobby 인스턴스는 한달에 7달러를 지불하게 된다.
이 결제를 BIYN 에서 결제가 가능한 방안을 마련하여야 한다.

**Comment by plainOldCode** [comment]
실제 결제가 이루어지는 시점에서 이슈가 발생할 것 같다.
Dyno 의 변경은 nodelay@gmail.com 계정으로 진행했지만 실제 결제도 그렇게 연결이 될까?

**Comment by jennybehan** [comment]
해당 결제가 저에게 이루어 진 것은 아닌지 궁금합니다.
Heroku에서 Application dynos로 1.45 달러가 청구되었는데 전에는 없던 결제라서 확인한 번 해주시면 감사하겠습니다.

**Comment by plainOldCode** [comment]
> 해당 결제가 저에게 이루어 진 것은 아닌지 궁금합니다.
> Heroku에서 Application dynos로 1.45 달러가 청구되었는데 전에는 없던 결제라서 확인한 번 해주시면 감사하겠습니다.

Dyno hobby 로 변경은 제가 했지만 소유자가 주연님으로 되어있어서 결제도 주연님 계정의 카드로 된 것으로 추측됩니다.
해당 결제 금액을 BIYN 운영팀에 요청해야할 것 같습니다.

그런데 앞으로도 이렇게 되면 조금 번거로울 것 같습니다. 
만약, BIYN 운영팀에서 해외 결제가 가능한 카드 번호가 있다면 다음의 방식은 어떠할까요?

1. tech@biyn.kr github 계정 생성
2. tech@biyn.kr heroku 초대 (카드 등록)
3. tech@biyn.kr 에 권한 이양

**Comment by jennybehan** [comment]
넵 말씀하신대로 공통 계정으로 만들어야 할 것 같습니다.
이번에는 BIYN에 일단 청구 후 위 방식으로 진행하면 될 것 같습니다.

**Comment by tech-biyn** [comment]
[x] tech@biyn.kr github 계정 생성
[x] tech@biyn.kr heroku 초대
[ ] tech@biyn.kr heroku 카드 등록
[ ] tech@biyn.kr heroku 권한 이양

**Comment by plainOldCode** [comment]
heroku 결제는 github private 전환과 관계 없이 공통계정에서 결제가 이루어져야 하지 않을까 합니다.

**Comment by jennybehan** [comment]
네 그렇게 하기로 한 거 아니었나요?

**Comment by plainOldCode** [comment]
네 그런데 이부분이 repo private 문제가 논의 되면서 고정비 문제가 나와 일단 다시 상기하고자 하였습니다.

**Comment by jennybehan** [comment]
아 넵 이런 외부 계정들은 전부 공통계정으로 전환해야 합니다.
